### PR TITLE
mgmt, local generation for ContainerService

### PR DESCRIPTION
**Pull request by v-hongli1**
# Description

No manual code.

Please add an informative description that covers that changes made by the pull request and link all relevant issues.

If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.

# All SDK Contribution checklist:
- [ ] **The pull request does not introduce [breaking changes]**
- [ ] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [ ] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md).**

## [General Guidelines and Best Practices](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md#developer-guide)
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md#building-and-unit-testing)
- [ ] Pull request includes test coverage for the included changes.

### CODE REVIEW

File: sdk/resourcemanager/azure-resourcemanager-containerservice/src/main/java/com/azure/resourcemanager/containerservice/implementation/KubernetesClusterNetworkProfileImpl.java [hunk 1]
```diff
@@ -50,8 +50,8 @@ public KubernetesClusterNetworkProfileImpl withDnsServiceIP(String dnsServiceIP)
     }
 
     @Override
+    @Deprecated
     public KubernetesClusterNetworkProfileImpl withDockerBridgeCidr(String dockerBridgeCidr) {
-        ensureNetworkProfile().withDockerBridgeCidr(dockerBridgeCidr);
         return this;
```
Comment (XiaofeiCao) [inline sdk/resourcemanager/azure-resourcemanager-containerservice/src/main/java/com/azure/resourcemanager/containerservice/implementation/KubernetesClusterNetworkProfileImpl.java:55]: Seems manual code here? You could put manual code part in a separate commit.

**Comment by weidongxu-microsoft** [comment]
Please either put manual code to a separate commit, or comment in PR where you manually changed.

It is hard to review on 83 files to find where we need to focus.

File: sdk/resourcemanager/azure-resourcemanager-containerservice/src/main/java/com/azure/resourcemanager/containerservice/models/KubernetesCluster.java [hunk 2]
```diff
@@ -369,6 +369,7 @@ interface NetworkProfileDefinitionStages {
              * Docker bridge network.
              *
              * @param <ParentT> the stage of the network profile definition to return to after attaching this definition
+             * @deprecated The property has no effect since 2019
```
Comment (weidongxu-microsoft) [inline sdk/resourcemanager/azure-resourcemanager-containerservice/src/main/java/com/azure/resourcemanager/containerservice/models/KubernetesCluster.java:372]: move it to the API of `@Deprecated` annotation

File: sdk/resourcemanager/azure-resourcemanager-containerservice/src/main/java/com/azure/resourcemanager/containerservice/implementation/KubernetesClusterNetworkProfileImpl.java (hunk 1 above)
Comment (weidongxu-microsoft) [reply sdk/resourcemanager/azure-resourcemanager-containerservice/src/main/java/com/azure/resourcemanager/containerservice/implementation/KubernetesClusterNetworkProfileImpl.java:55]: There should be a getter as well?

File: sdk/resourcemanager/azure-resourcemanager-containerservice/src/main/java/com/azure/resourcemanager/containerservice/models/KubernetesCluster.java (hunk 2 above)
Comment (v-hongli1) [reply sdk/resourcemanager/azure-resourcemanager-containerservice/src/main/java/com/azure/resourcemanager/containerservice/models/KubernetesCluster.java:372]: Is change to `@Deprecated(since = "The property has no effect since 2019")` ? This description is inconsistent with the existing description. For example:
image

Comment (v-hongli1) [reply sdk/resourcemanager/azure-resourcemanager-containerservice/src/main/java/com/azure/resourcemanager/containerservice/models/KubernetesCluster.java]: Ok, fixed in the new version.

File: sdk/resourcemanager/azure-resourcemanager-containerservice/CHANGELOG.md [hunk 3]
```diff
@@ -2,13 +2,13 @@
 
 ## 2.27.0-beta.1 (Unreleased)
 
-### Features Added
-
 ### Breaking Changes
 
-### Bugs Fixed
+- The property `dockerBridgeCidr` has no effect since 2019.
```
Comment (weidongxu-microsoft) [inline sdk/resourcemanager/azure-resourcemanager-containerservice/CHANGELOG.md:7]: Include the class

```suggestion
- The property `dockerBridgeCidr` in class `KubernetesCluster` has no effect since 2019.
```

File: sdk/resourcemanager/azure-resourcemanager-containerservice/pom.xml [hunk 4]
```diff
@@ -42,6 +42,7 @@
 
       --add-opens com.azure.core/com.azure.core.implementation.util=ALL-UNNAMED
     </javaModulesSurefireArgLine>
+    <revapi.skip>true</revapi.skip>
```
Comment (weidongxu-microsoft) [inline sdk/resourcemanager/azure-resourcemanager-containerservice/pom.xml:45]: What breaks?

Comment (v-hongli1) [reply sdk/resourcemanager/azure-resourcemanager-containerservice/pom.xml:45]: Please refer the [check results](https://dev.azure.com/azure-sdk/public/_build/results?buildId=2797131&view=results)

File: sdk/resourcemanager/azure-resourcemanager-containerservice/src/main/java/com/azure/resourcemanager/containerservice/implementation/KubernetesClusterNetworkProfileImpl.java (hunk 1 above)
Comment (v-hongli1) [reply sdk/resourcemanager/azure-resourcemanager-containerservice/src/main/java/com/azure/resourcemanager/containerservice/implementation/KubernetesClusterNetworkProfileImpl.java:55]: `KubernetesClusterNetworkProfileImpl` is an interface implementation class that must implement all methods in the interface, even if the method is marked as "deprecated".

File: sdk/resourcemanager/azure-resourcemanager-containerservice/pom.xml (hunk 4 above)
Comment (weidongxu-microsoft) [reply sdk/resourcemanager/azure-resourcemanager-containerservice/pom.xml:45]: that is a cancelled pipeline. nothing shows. If you have it, paste here.

Comment (XiaofeiCao) [reply sdk/resourcemanager/azure-resourcemanager-containerservice/pom.xml:45]: I can't see the check results. Could you share screenshots?

File: sdk/resourcemanager/azure-resourcemanager-containerservice/src/main/java/com/azure/resourcemanager/containerservice/implementation/KubernetesClusterNetworkProfileImpl.java (hunk 1 above)
Comment (XiaofeiCao) [reply sdk/resourcemanager/azure-resourcemanager-containerservice/src/main/java/com/azure/resourcemanager/containerservice/implementation/KubernetesClusterNetworkProfileImpl.java:55]: Strangely there's no corresponding getter method defined in the interface for this property. Guess we're good.

Comment (weidongxu-microsoft) [reply sdk/resourcemanager/azure-resourcemanager-containerservice/src/main/java/com/azure/resourcemanager/containerservice/implementation/KubernetesClusterNetworkProfileImpl.java:55]: I got it. The getter is `ContainerServiceNetworkProfile networkProfile()`

File: sdk/resourcemanager/azure-resourcemanager-containerservice/pom.xml (hunk 4 above)
Comment (v-hongli1) [reply sdk/resourcemanager/azure-resourcemanager-containerservice/pom.xml]: Please refer to the [check results](https://dev.azure.com/azure-sdk/public/_build/results?buildId=2797271&view=results)

Comment (weidongxu-microsoft) [reply sdk/resourcemanager/azure-resourcemanager-containerservice/pom.xml]: OK, it should be OK to skip.

Remember, DO NOT skip it, before you have that report, and have it reviewed with reviewers.

**Review by weidongxu-microsoft — APPROVED** [review]

File: sdk/resourcemanager/azure-resourcemanager-containerservice/pom.xml (hunk 4 above)
Comment (weidongxu-microsoft) [reply sdk/resourcemanager/azure-resourcemanager-containerservice/pom.xml:45]: @XiaofeiCao I am a bit surprised that the break on `ContainerServiceNetworkProfile` does not flag as break on azure-resourcemanager (as KubernetesCluster exposes it as return of a getter).

But if it not get flagged, we would be good.

**Review by XiaofeiCao — APPROVED** [review]
